modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -48,12 +48,19 @@ spec:
   - EvictPodsWithLocalStorage
   - EvictPodsWithPVC
 ----
-<1> Optional: Set a list of user-created namespaces to include or exclude from descheduler operations. Use `excluded` to set a list of namespaces to exclude or use `included` to set a list of namespaces to include. Note that protected namespaces (`openshift-*`, `kube-system`, `hypershift`) are always excluded.
+--
+<1> Optional: Set a list of user-created namespaces to include or exclude from descheduler operations. Use `excluded` to set a list of namespaces to exclude or use `included` to set a list of namespaces to include. Note that protected namespaces (`openshift-*`, `kube-system`, `hypershift`) are excluded by default.
++
+[IMPORTANT]
+====
+The `LowNodeUtilization` strategy does not support namespace exclusion. If the `LifecycleAndUtilization` profile is set, which enables the `LowNodeUtilization` strategy, then no namespaces are excluded, even the protected namespaces. If you want to avoid evictions from the protected namespaces while the `LowNodeUtilization` strategy is enabled, then you must set the priority class name to `system-cluster-critical` or `system-node-critical`.
```

Comment:
For minimalism, you might want to change to "To avoid evictions..."

Reply:
Good suggestion, thanks!